pulldown-cmark-escape/src/lib.rs

```diff
@@ -71,37 +74,64 @@ where
     }
 }
 
+/// This wrapper exists because we can't have both a blanket implementation
+/// for all types implementing `Write` and types of the for `&mut W` where
```

Comment:
This should be disambiguated to refer to `io::Write` / `fmt::Write`. Also a typo (seems to exist in the old comment as well):
```suggestion
/// for all types implementing `io::Write` and types of the form `&mut W` where
```